test/test.js

```diff
@@ -11,3 +11,16 @@ test('defaults', async t => {
     t.pass();
   }
 });
+
+test('invalid dashboard name', async t => {
+  const stackName = cfntest.stackName();
+  try {
+    t.log(await cfntest.createStack(`${__dirname}/invalid-dashboard-name.yml`, stackName, {}));
+    // what could we test here?
+    t.log(await cfntest.deleteStack(stackName));
```

Comment:
please move the deleteStack into finally

Reply:
👍